### PR TITLE
fix #191201

### LINKED ISSUE #191201: VSCode open a terminal in Terminal Panel before opening a terminal in a tab in the editor window
Type: <b>Bug</b>

First, add this setting in your config:

`"terminal.integrated.defaultLocation": "editor"`

Then, I have tasks that I can run with shortcuts. For instance:
```
{
       "label": "debug terminal issue",
       "type": "shell",
       "command": "echo 'Hello World!'"
 }
```

When running the task, it will first open a terminal in the Terminal Panel, and then open a new tab with the terminal result. However, I unticked "Terminal" from the list of tabs that should be in the Terminal Panel. Why does it keep opening the Terminal Panel with an empty terminal before opening a new terminal tab in the editor window, where the task is running? I assume this is not a wanted solution.

VS Code version: Code 1.81.1 ("6c3e3dba23e8fadc360aed75ce363ba185c49794", 2023-08-09T22:22:42.175Z)
OS version: Windows_NT x64 10.0.22621

### PR DESCRIPTION
**Pull request by tisilent**
fix #191201

### CODE REVIEW

File: src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts [hunk 1]
```diff
@@ -1045,7 +1045,9 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 			this._viewsService.openView(Markers.MARKERS_VIEW_ID);
 		} else if (task.command.presentation && (task.command.presentation.focus || task.command.presentation.reveal === RevealKind.Always)) {
 			this._terminalService.setActiveInstance(terminal);
-			this._terminalGroupService.showPanel(task.command.presentation.focus);
+			if (terminal.target !== TerminalLocation.Editor) {
```
Comment (meganrogge) [inline src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts:1048]: let's call this instead

https://github.com/microsoft/vscode/blob/95f948717b45f8ee70672092bd93953cb2551744/src/vs/workbench/contrib/terminal/browser/terminalService.ts#L556-L566

**Review by meganrogge — CHANGES_REQUESTED** [review]
Thanks for working on this. Just a suggestion

File: src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts (hunk 1 above)
Comment (tisilent) [reply src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts:1048]: I see.

**Review by meganrogge — APPROVED** [review]
Thanks!

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**